### PR TITLE
Add Radio component

### PR DESCRIPTION
**Pull request by benaube**
Add Radio component
Add ratio tool to ToolBar

### CODE REVIEW

File: src/components/MaterialUI/Inputs/Radio.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+			position: "absolute",
+			borderRadius: "50%",
+			top: "50%",
+			left: "50%",
+			transform: "translate(-50%, -50%)",
+		},
+	},
+}));
+
+const maybeTranslate = (formatMessage, message) => (message?.id ? formatMessage(message) : message);
```
Comment (joclefrancois) [inline src/components/MaterialUI/Inputs/Radio.js:41]: Do we have such pattern somewhere else? Should we not assume that all labels will already be localized like in the Checkbox or the Table components ? I would say it is the responsibility of the component user to make sure the label is set properly.

Comment (benaube) [reply src/components/MaterialUI/Inputs/Radio.js:41]: In the OMS Toolbar, nothing was translated. Both the button and input are translated in orc-shared.
Input.js does the MaybeTranslate.
IconButton uses Text.js which does the translation.

File: src/components/MaterialUI/Inputs/Radio.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+				value={value}
+				onChange={handleChange}
+			>
+				{radios.map((radio, index) => {
+					const handleClick = radio.clickEvent ? event => radio.clickEvent(event.target.value) : null;
+
+					return (
+						<FormControlLabel
+							key={index}
+							name={radio.value}
```
Comment (joclefrancois) [inline src/components/MaterialUI/Inputs/Radio.js:94]: I am guessing radio.value should be unique in the collection of radios. Did you check that out? If yes, I would also use it as a key with a prefix, something like "radiobutton_" + radio.value. It is usually not recommended to use index as a key, I agree that sometimes it's hard to use something else but whenever we can, we should use something else.

File: src/components/MaterialUI/Inputs/Radio.js (hunk 1 above)
Comment (joclefrancois) [reply src/components/MaterialUI/Inputs/Radio.js:41]: I would say that this is not the approach we take for Material-UI wrapper components we create. We want to eventually deprecate Gert's components you are referring to. I understand you point though but we should keep the same recipe for all of our wrappers. I am not saying the current one is the best one, I am just saying we should keep consistency throughout our code when related to Material-UI. Even Gert's toolbar might be replaced eventually.

Comment (benaube) [reply src/components/MaterialUI/Inputs/Radio.js:41]: Alright, so you'd rather Radio doesn't translate and i do the translation in OMS useConfig where the tools are defined?

Comment (joclefrancois) [reply src/components/MaterialUI/Inputs/Radio.js:41]: I would say yes, especially that the label can also be a component or something else. I wouldn't give the responsibility of translation to our components.

**Comment by joclefrancois** [comment]
Look at the coverage using the following link, if not a big deal, I would try to add or change a unit test to have coverage at 100%.

https://travis-ci.com/github/Orckestra/orc-shared/jobs/363386869

**Pull request merged**